### ISSUE TITLE
"New" method

### ISSUE THREAD

**Issue by jwindha1** [bfelbo/DeepMoji#30]
Hello,

I am unsure about the meaning of the "new" method for transfer learning on DeepMoji. In finetuning.py it seems that the only difference between "full" and "new" is the loss rate. The paper describes "new" as a "model trained without pretraining," indicating that it is similar to a network with the same architecture but without transferring any weights. However, when I run "new" against both "full" and a newly instantiated model (using deepmoji_architecture), I get differing results. I would very much appreciate an explanation. Thank you!

**Comment by bfelbo** [comment]
For the `new` method you should use leave the pretrained weight path as `None` (see https://github.com/bfelbo/DeepMoji/blob/master/deepmoji/model_def.py#L73). Might that be the cause of the difference you're talking about?

Also, note that there'll always be minor differences between runs.

**Comment by bfelbo** [comment]
Please feel free to reopen the issue if you're still having problems.
